.github/workflows/deploy-affected.yml

```
@@ -0,0 +1,34 @@
+name: Deploy Affected Apps (Disabled)
```

actionlint (1.7.11)

[error] "on" section is missing in workflow

(syntax-check)